### ISSUE TITLE
Basemap error won't resolve even after compliance

### ISSUE THREAD

**Issue by HippyWho** [domlysz/BlenderGIS#149]
Whenever I click on Basemap or hit the * key I get this error:

> "No imaging library available. Please install Python GDAL or Pillow module."

So I installed Pillow with this confirmation from Terminal, but the error still appears.

> /Library/Frameworks/Python.framework/Versions/3.6/bin/pip3 install pillow
Collecting pillow
  Downloading https://files.pythonhosted.org/packages/81/96/05a5c9ba4a75ed330234780e4ae30018bbf77c847982ff6a16b8b291a0da/Pillow-5.4.1-cp36-cp36m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl (3.7MB)
    100% |████████████████████████████████| 3.7MB 6.2MB/s 
Installing collected packages: pillow
Successfully installed pillow-5.4.1

In the directory noted therein lots of files with "gdal" were installed. So by all measures it seems I've done what the error says must be done, yet the error keeps popping up when I click on Basemap or hit the * key. I've closed and reopened Blender several time in case it needs to refresh its look at Python, but no luck. 

Anyone know how to get past this error? 

I'm on a Mac desktop, OS X El Capitan v 10.11.6 with Blender 2.79b and QGIS 3.

**Comment by domlysz** [comment]
By default BlenderGIS rely on [imageio ](https://github.com/imageio/imageio) library, a common Python interface to various image processing tool. [Freeimage ](http://freeimage.sourceforge.net/) is the most basic toolset used throught the imageio lib. When BlenderGIS addon is enable, Freeimage binary files are automatically downloaded. Freeimage provides most of image processing functionnalities needed by BlenderGIS. It seems in your case that Freeimage cannot be correctly installed and is not available on your system. In my code, getting Freeimage can fails silently (shame on me), I must do make an update to correctly raise the error in this case.

I've no knowledge about MAC OS, but it seems like in Windows Blender bundle it's own Python installation, so molules installed with pip are not available in Blender Python environnement. Seems you can try to copy paste Python files in Blender folder tree, some clues [here](https://github.com/domlysz/BlenderGIS/wiki/How-to-install-GDAL#mac-osx)

Sorry if it's not help a lot

**Comment by HippyWho** [comment]
Thanks! Following those clues you link to I installed Xcode, but my luck ends there. The command "sudo port install gdal py34-gdal" returns: "sudo: port: command not found." But I found the 'osgeo' folder in Python labyrinth and copied it into my Blender directory, but which has no subfolders as indicated in the path given: /Blender/blender.app/Contents/Resources/2.74/scripts/modules/. So I'm perplexed as to why Blender on my system only has an app file. The BlenderGIS folder is much more complex, with four subfolders, even a nested folder named basemaps, alas.

I'm able to bring basemaps into QGIS. Can't I bring a QGIS file with basemaps into Blender?
